app/routes/farmer-apply/confirmation.js

```diff
@@ -13,18 +13,21 @@ module.exports = {
       // TODO: Get this data based on eligibility or the applicant
       // const { sbi } = request.payload
       const organisation = session.getOrganisation(request)
-
       // TODO: should the reference number be a particular format?
-      const reference = uuid().split('-').shift().toLocaleUpperCase('en-GB')
+      const reference = uuidv4().split('-').shift().toLocaleUpperCase('en-GB')
```

Comment:
With the reference being created within the `application` there shouldn't be a need to do this. If so, this line, line 18 and line 1 should be ok to delete.